cmd/juju/cloud/add.go

```diff
@@ -137,6 +148,14 @@ func (c *addCloudCommand) runInteractive(ctxt *cmd.Context) error {
 		return errors.Trace(err)
 	}
 
+	pollster.VerifyURLs = func(s string) (ok bool, msg string, err error) {
+		err = c.Ping(provider, s)
```

Comment:
Well, they're specifically not supported.  You can't interactively add-cloud for AWS, it's not an option we'll let you pick, so we won't ever see not implemented.